bench/workgen/runner/prepare_stress.py

```diff
@@ -0,0 +1,206 @@
+statistics_log=(wait=1,json)"
+
+# 1B records * (key=12 + value=138) is about 150G total data size
+key_sz=12
+value_sz=138
+log_like_table=true
+table_config="type=file"
+icount=1000000000
+report_interval=5
+run_time=3600
+# Scans every 10 minutes for all the scan specific tables.
+# .4B records * (key=12 + value=138) is about 60G total data size for scan
+# Running on a machine with 64G physical memory, this exhausts both the
+# WT cache and the system cache.
+scan_interval=600
+scan_pct=100
+scan_table_count=20
+scan_icount=400000000
+populate_threads=5
+table_count=100
+threads=((count=400,reads=1),(count=20,inserts=1,throttle=500),(count=10,updates=1,throttle=500))
+# Add throughput/latency monitoring
+max_latency=50000
+sample_interval=5
+'''
+###################################################################################################
+
+from runner import *
+from wiredtiger import *
+from workgen import *
+import time
+
+context = Context()
+conn_config =   "cache_size=1G,checkpoint=(wait=60,log_size=2GB),\
+                eviction=(threads_min=12,threads_max=12),log=(enabled=true),session_max=800,\
+                debug_mode=(table_logging=true),\
+                eviction_target=60,statistics=(fast),statistics_log=(wait=1,json)"# explicitly added
+conn = context.wiredtiger_open("create," + conn_config)
+s = conn.open_session("")
+
+wtperf_table_config = "key_format=S,value_format=S," +\
+    "exclusive=true,allocation_size=4kb," +\
+    "internal_page_max=64kb,leaf_page_max=4kb,split_pct=100,"
+compress_table_config = ""
+table_config = "type=file"
+tables = []
+table_count = 1
+for i in range(0, table_count):
+    tname = "table:test" + str(i)
+    table = Table(tname)
+    s.create(tname, wtperf_table_config +\
+             compress_table_config + table_config)
+    table.options.key_size = 200
+    table.options.value_size = 5000
+    tables.append(table)
+
+populate_threads = 40
+icount = 500000
+
+start_time = time.time()
+
+# If there are multiple tables to be filled during populate,
+# the icount is split between them all.
+pop_ops = Operation(Operation.OP_INSERT, tables[0])
+pop_ops = op_multi_table(pop_ops, tables)
+nops_per_thread = icount // (populate_threads * table_count)
+pop_thread = Thread(pop_ops * nops_per_thread)
+pop_workload = Workload(context, populate_threads * pop_thread)
+pop_workload.run(conn)
+
+# Log like file, requires that logging be enabled in the connection config.
+log_name = "table:log"
+s.create(log_name, wtperf_table_config + "key_format=S,value_format=S," +\
+        compress_table_config + table_config + ",log=(enabled=true)")
+log_table = Table(log_name)
+
+# Read operation with read_timestamp_lag
+ops = Operation(Operation.OP_SEARCH, tables[0],Key(Key.KEYGEN_PARETO, 0, ParetoOptions(1)))
+ops = op_multi_table(ops, tables, False)
+ops = op_log_like(ops, log_table, 0)
+read_txn = txn(ops, 'read_timestamp')
+read_txn.transaction.read_timestamp_lag = 2
+thread0 = Thread(read_txn)
```

Comment:
We discussed changing the ordering here, so that the transaction is around a small piece of work, rather than many tables.  Something like:
```
ops = Operation(Operation.OP_SEARCH, tables[0],Key(Key.KEYGEN_PARETO, 0, ParetoOptions(1)))
ops = txn(ops, 'read_timestamp')
ops.transaction.read_timestamp_lag = 2
ops = op_multi_table(ops, tables, False)
ops = op_log_like(ops, log_table, 0)
thread0 = Thread(ops)
```
Please make this change for all the operations that use the `txn` function.